src/ansi-c/c_typecheck_code.cpp

```diff
@@ -903,39 +922,91 @@ void c_typecheck_baset::typecheck_spec_assigns_condition(exprt &condition)
     throw 0;
 }
 
+void c_typecheck_baset::typecheck_conditional_targets(
+  exprt::operandst &targets,
+  const std::function<void(exprt &)> typecheck_target,
+  const std::string &clause_type)
+{
+  exprt::operandst tmp;
+  bool must_throw = false;
+
+  for(auto &target : targets)
+  {
+    if(!can_cast_expr<conditional_target_group_exprt>(target))
+    {
+      must_throw = true;
+      error().source_location = target.source_location();
+      error() << "expected ID_conditional_target_group expression in " +
```

Comment:
The syntax section of the manual defines a conditional target group as :

```
conditional-target-group ::= (condition ':')? target (',' target)*
``` 

How about using `conditional target group` instead of `ID_conditional_target_group` to make it more human-readable ?

Reply:
I like it!